docs/csharp/language-reference/keywords/interpolated-strings.md

```diff
@@ -17,12 +17,12 @@ Used to construct strings.  An interpolated string looks like a template string
 The arguments of an interpolated string are easier to understand than a [composite format string](../../../standard/base-types/composite-formatting.md#composite-format-string).  For example, the interpolated string  
   
 ```csharp  
-Console.WriteLine($"Name = {name}, hours = {date:hh}"); 
+Console.WriteLine($"Name = {name}, hours = {hours}:hh");
```

Comment:
This change introduced a coding error:

```csharp
{hours}:hh
```

should be:

```csharp
{hours:hh}
```

Otherwise, the variable won't be formatted correclty.